### PR TITLE
docs for allow-priviledged flag

### PR DESCRIPTION
**Pull request by alexcb**
Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

**Comment by alexcb** [comment]
@vladaionescu how did you manage to get `https://docs.earthly.dev/docs/earthfile#privileged` to point to the first one? in my case referencing `#allow-privileged` is pointing to the second anchor.

File: docs/earthfile/earthfile.md [hunk 1]
```diff
@@ -1018,6 +1066,13 @@ If an `IMPORT` is defined in the `base` target of the Earthfile, then it becomes
 
 For more information see the [target, artifact and command references guide](../guides/target-ref.md).
 
+#### Options
+
+##### `--allow-privileged`
+
+Same as [`FROM --allow-privileged`](#allow-privileged).
```
Comment (vladaionescu) [inline docs/earthfile/earthfile.md:1073]: I would expand a little more here, given that this is the `IMPORT` command. We should say that it allows any mention of the alias to use privileged.

Comment (alexcb) [reply docs/earthfile/earthfile.md]: That's a good call, how does it sound now?

**Review by vladaionescu — APPROVED** [review]
:+1:

**Pull request merged**